### PR TITLE
Pim 5437 - Makes Akeneo PIM CE 1.2 compatible with DCI

### PR DESCRIPTION
**Pull request by BitOne**
| Q                    | A
| -------------------- | ---
| Bug fix?             | No
| New feature?         | No
| BC breaks?           | No
| CI currently passes? | Yes
| Tests pass?          | Yes
| Scenarios pass?      | Yes
| Checkstyle issues?*  | No 
| PMD issues?**        | No 
| Changelog updated?   | N.A.
| Fixed tickets        | PIM-5437
| Doc PR               | No

Please note the JUnitFormatter comes from master.

### CODE REVIEW

**Comment by jjanvier** [comment]
gtm

File: features/product/toggle_status_product.feature [hunk 1]
```diff
@@ -9,6 +9,7 @@ Feature: Toggle status of a product
     And a "CD player" product
     And I am logged in as "Julia"
 
+  @unstable
   Scenario: Disable product from grid
```
Comment (ClementGautier) [inline features/product/toggle_status_product.feature]: Is this scenario unstable on >=1.3 too ? If yes GTM, if not please fix the behat insetad of skipping it.

**Comment by juliensnz** [comment]
GTM

File: features/product/toggle_status_product.feature (hunk 1 above)
Comment (BitOne) [inline features/product/toggle_status_product.feature]: Scenario is not fixed or mark unstable on version >= 1.3. But as it's based on flash message, it's definitely unstable.

**Pull request merged**